### DISCUSSION TITLE
When 3.11.4 will be released?

### DISCUSSION THREAD

**Discussion by aiyolo** [nlohmann/json#4576] [General]
There are some convinient features in 3.11.4 that will really simplify my work a lot, I just wonder that if there is a plan to release 3.11.4?

**Comment by nlohmann** [comment]
The short answer: When it's done.

The long answer: There is currently no plan what needs to be finished before the release. At some point early next year, I would start writing the release notes to get an overview what was changed, and what would make sense to extend/test/document before the release.

As always, PRs and sponsorships welcome. :-)
